.devcontainer/setup.sh

```diff
@@ -0,0 +1,41 @@
+# install curl, git, ...
```

Comment:
Actually, I think all of this can and should be done in the dockerfile. No need for an additional script.